SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -22,13 +24,30 @@ FString USpatialGDKSettings::ToString()
 	Args.Add(HeartbeatIntervalSeconds);
```

Comment:
I believe we can get rid of the `ToString` function completely as it's never used.

Reply:
Yes, I did this on my local branch without any impact